spec.md

```diff
@@ -377,24 +377,32 @@ how Platforms might expose these values to their users.
 
 | Response field | Type | Description |
 | --- | --- | --- |
-| [create](#input-parameters-object) | object | The schema definition for creating a Service Instance. |
-| [update](#input-parameters-object) | object | The schema definition for updating a Service Instance. |
+| [create](#json-schema-objects) | object | The schema definition for creating a Service Instance. |
+| [update](#json-schema-objects) | object | The schema definition for updating a Service Instance. |
 
 
 ##### Service Binding Object
 
 | Response field | Type | Description |
 | --- | --- | --- |
-| [create](#input-parameters-object) | object | The schema definition for creating a Service Binding. |
+| [create](#json-schema-objects) | object | The schema definition for creating a Service Binding. |
 
 
-##### Input Parameters Object
+##### JSON Schema Objects
+
+It is useful for the Platform for the Broker to define exactly what is expected
```

Comment:
Is "Platform" here correct? I don't think you really mean "Platform" do you?  Perhaps just "It is useful for the Broker to define..."

Reply:
It is not useful for the broker. I mean controller in the k8s side, but I think in the OSB spec we call that the `Platform`. So, yes I think I do mean `Platform` here. I want to say it is useful for the caller, the consumer of the broker API.

Reply:
ok, can I suggest a rewording then since the "for... for..." confused me.
How about: `It is useful for the Platform if the Broker defines exactly what is expected...`  ?

Reply:
how about `A Platform will find it useful for the Broker to define`